docs/developer/DEVELOP.md

```diff
@@ -39,6 +44,9 @@ RH_CERT_URL="https://foo.bar"
 # enable Bugzilla backwards sync to propagate writes to Bugzilla
 # otherwise all the writes are performed only locally in OSIDB
 BBSYNC_SYNC_TO_BZ=1
+
+# Enable https proxy (optional)
+HTTPS_PROXY="http://foo.bar"
```

Comment:
Could there by a short explanation that it is required for the Jira stage and maybe a real default value. Otherwise I am sure I would forget the meaning and the value and would have to ask you the next time :smile:

Reply:
I have added a better description in `DEVELOP.md` and added in taskman constant file default values for `JIRA_TASKMAN_PROJECT_KEY`, `JIRA_TASKMAN_URL` and `HTTPS_PROXY`